app/views/customers/show.html.erb

```diff
@@ -0,0 +1,20 @@
+<div>
+  <h1> <%= @customer.name%> Show Page </h1>
+  <h2> __Items__ </h2>
+  <% @customer.items.each do |item| %>
+    <p> 
```

Comment:
You can add the css id to this <p> element to target your within blocks for testing.